demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -139,6 +139,142 @@ class DemoTileOrganizer {
+    (this.querySelector('.video-tile-pause') as HTMLElement).style.display = display;
+  }
+
+  public set showRemoteVideoPreferences(shouldShow: boolean) {
+    const display: string = shouldShow? 'block' : 'none';
+    (this.querySelector('.target-resolution-header') as HTMLElement).style.display = display;
+    (this.querySelector('.target-resolution-toggle') as HTMLElement).style.display = display;
+    (this.querySelector('.video-priority-header') as HTMLElement).style.display = display;
+    (this.querySelector('.video-priority-toggle') as HTMLElement).style.display = display;
+  }
+
+  show(isContent: boolean) {
+    this.classList.add(`active`);
+    if (isContent) {
+      this.classList.add('content');
+    }
+  }
+
+  public set featured(featured: boolean) {
+    if (featured) {
+      this.classList.add(`featured`);
+    } else {
+      this.classList.remove(`featured`);
+    }
+  }
+
+  hide() {
+    this.classList.remove('active', 'featured', 'content');
+  }
+
+  showVideoStats(
+    keyStatstoShow: { [key: string]: string },
+    metricsData: { [id: string]: {[key: string]: number} },
+    streamDirection: string,
+  ): void {
+    const streams = metricsData ? Object.keys(metricsData) : [];
+    if (streams.length === 0) {
+      return;
+    }
+
+    let statsInfo: HTMLDivElement = this.querySelector(`#stats-info`) as HTMLDivElement;
+    if (!statsInfo) {
+      statsInfo = document.createElement('div');
+      statsInfo.setAttribute('id', `stats-info`);
+      statsInfo.setAttribute('class', `stats-info`);
+    }
+
+    let statsInfoTable = this.querySelector(`#stats-table`) as HTMLTableElement;
+    if (statsInfoTable) {
+      statsInfo.removeChild(statsInfoTable);
+    }
+    statsInfoTable = document.createElement('table') as HTMLTableElement;
+    statsInfoTable.setAttribute('id', 'stats-table');
+    statsInfoTable.setAttribute('class', 'stats-table');
+    statsInfo.appendChild(statsInfoTable);
+
+    this.videoElement.insertAdjacentElement('afterend', statsInfo);
+    const header = statsInfoTable.insertRow(-1);
+    let cell = header.insertCell(-1);
+    cell.innerHTML = 'Video statistics';
+    for (let cnt = 0; cnt < streams.length; cnt++) {
+      cell = header.insertCell(-1);
+      cell.innerHTML = `${streamDirection} ${cnt + 1}`;
+    }
+
+    for (const ssrc of streams) {
+      for (const [metricName, value] of Object.entries(metricsData[ssrc])) {
+        if (keyStatstoShow[metricName]) {
+          const rowElement = document.getElementById(
+            `${metricName}`
+          ) as HTMLTableRowElement;
+          const row = rowElement ? rowElement : statsInfoTable.insertRow(-1);
+          if (!rowElement) {
+            row.setAttribute('id', `${metricName}`);
+            cell = row.insertCell(-1);
+            cell.innerHTML = keyStatstoShow[metricName];
+          }
+            cell = row.insertCell(-1);
+            cell.innerHTML = `${value}`;
```

Comment:
Indenting.

Reply:
Ergh, need to get the auto formatter/linter to run on demo as well.  todo....